### DISCUSSION TITLE
Server Desync on death

### DISCUSSION THREAD

**Discussion by superryan14** [Pryaxis/TShock#1799] [Q&A]
I just started using tshock to be able to disable some stuff in game. it works great for a good 20~ minutes but sometimes on death I will desync with my own server, I can't open chests, can't teleport with pylons and anyone near me at spawn doesn't move. Same thing happens to other players on the server as well. I only encounter this problem with tshock so i'm not sure what to do to fix this

**Comment by TheCipher01** [comment]
I'm also having this issue. Or at least it seems to be that. My friends and I started noticing some of these desync issues. Just tested it by dying and I was unable to open any chests and my friends don't see me moving. Relogging seems to fix the issue.

**Reply by superryan14** [reply]
Yeah, reloging fixes it, so every time I die I have to rejoin the server to fix everything

**Comment by TheCipher01** [comment]
I should also add I'm experiencing this without using SSC

**Reply by TheCipher01** [reply]
I've gone ahead an made this an issue report (#1800 ) since it seems that this is a bit more issue/bug report worthy.
